package.json

```diff
@@ -51,7 +53,7 @@
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.25.1",
     "eslint-plugin-react-hooks": "^4.2.0",
-    "husky": "4.3.8",
+    "husky": "7.0.2",
```

Comment:
@IvanSanchezSalvador did you use `yarn upgrade husky --latest` or `yarn add husky@7.0.2` for this? it looks like the lockfile wasn't updated ofr this change which is creating a problem (as we can see in the failed logs)

can you try running one of those 2 commands and adding that lockfile change to the PR?